app/src/main/res/values/strings.xml

```diff
@@ -600,6 +600,14 @@
     <string name="preference_option_autoplay_blocked">Video and audio blocked</string>
     <!-- Label that indicates video and audio autoplay is allowed -->
     <string name="preference_option_autoplay_allowed">Video and audio allowed</string>
+    <!-- Label that indicates that all video and audio autoplay is allowed -->
+    <string name="preference_option_autoplay_allowed2">Allow audio and video</string>
+    <!-- Label that indicates that video and audio autoplay is only allowed over WIFI -->
+    <string name="preference_option_autoplay_allowed_wifi_only">Allow audio and video on WIFI only</string>
```

Comment:
@Baron-Severin 
Why is `Wi-Fi` spelled `WIFI`? As far as I'm aware, that's not a form we use it anywhere
https://design.firefox.com/photon/copy/word-list.html#w

Did these go through the copy team? If I understand it correctly, Betsy should own content on mobile these days.

Reply:
`Wi-Fi` is correct.

Reply:
I was [asked](https://github.com/mozilla-mobile/fenix/issues/8503#issuecomment-592658604) to use the strings in the mocks.  I'll update that string.